spec/spec_helper.rb

```diff
@@ -5,6 +5,7 @@
 require 'dotenv'
 require 'capybara/rspec'
 require 'capybara-webkit'
+require 'faker'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.